sorc/build_tropcy_NEMS.sh

```diff
@@ -29,6 +29,11 @@ else
 fi
 
 source ../modulefiles/modulefile.storm_reloc_v6.0.0.$target
+if [ $target = "wcoss2" ]; then
+  SIGIO_LIB4=$SIGIO_LIB
```

Comment:
Ibid

Reply:
Needed to support both stack (WCOSS2) and pre-stack (elsewhere).